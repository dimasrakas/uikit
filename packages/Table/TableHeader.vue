<template>
  <th>
    <slot />
  </th>
</template>

<script>
export default {
  name: "TableHeader",
};
</script>
