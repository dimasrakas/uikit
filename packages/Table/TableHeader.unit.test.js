import { shallowMount } from "@vue/test-utils";
import TableHeader from "./TableHeader.vue";

describe("TableHeader", () => {
  it('should have "TableHeader" as name', () => {
    expect(TableHeader.name).toBe("TableHeader");
  });

  it("should render correctly", () => {
    const wrapper = shallowMount(TableHeader);
    expect(wrapper.element).toMatchSnapshot();
  });
});
