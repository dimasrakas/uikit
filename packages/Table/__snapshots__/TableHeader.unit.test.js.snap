// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeader should render correctly 1`] = `<th />`;
